task/oci-copy/0.1/oci-copy.yaml

```diff
@@ -274,50 +281,39 @@ spec:
           name: varlibcontainers
       workingDir: $(workspaces.source.path)
     - name: sbom-generate
-      image: quay.io/konflux-ci/yq:latest@sha256:93bb15cff64b708263055a5814b24a0b450d8724b86a7e5206396f25d81fcc21
+      image: quay.io/acmiel-test/sbom-utility-scripts:spdx-support
```

Comment:
This will needs to change after https://github.com/konflux-ci/build-tasks-dockerfiles/pull/226, of course